app/models/professional_body_membership.rb

```diff
@@ -0,0 +1,3 @@
+class ProfessionalBodyMembership < ApplicationRecord
+  belongs_to :jobseeker_profile, optional: true
```

Comment:
is this correct? I would expect it to always be attached to a profile? Isn't that the idea?